docs/components/tooltips.md

```diff
@@ -0,0 +1,133 @@
+<!--docs:
+title: "Tooltips"
+layout: detail
+section: components
+excerpt: "Tooltips display informative text when users hover over, focus on, or tap an element."
+iconId: slider
+path: /catalog/Tooltips/
+-->
+
+# Tooltips
+
+[Tooltips](https://material.io/components/tooltips) display informative text when users hover over, focus on, or tap an element.
+
+![Tooltip example in a text editor app](assets/tooltips/Tooltips_hero.png)
+
+## Contents
+
+- [Using tooltips](#using-tooltips)
+  - [Making tooltips accessible](#making-tooltips-accessible)
+  - [Key properties for tooltips](#key-properties-for-tooltips)
```

Comment:
Remove these two headers. We only want the primary headers. 

I think you are using vs code which automatically generates all the headers so it might come back again

Reply:
Oh I added them because radio_buttons.md has them, but most of the other .md do not, I will remove here